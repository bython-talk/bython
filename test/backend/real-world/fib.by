# RUN: %filecheck $(basename %s .by) < $(%driver-full --inpath %s)
# CHECK: 0\n1\n1\n2\n3\n5\n8\n13\n21\n34
def fibonacci(n: i64) {
    if n == 0 || n == 1 {
        return n;
    };

    return n + fibonacci(n - 1);
}