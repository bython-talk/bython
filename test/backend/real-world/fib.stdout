0\n1\n1\n2\n3\n5\n8\n13\n21\n34